task18/index.js

```diff
@@ -0,0 +1,183 @@
+const URL = "https://mocki.io/v1/af7d1b29-da7d-4eba-977b-cdb5caeb2fff";
```

Comment:
```suggestion
const endpoint = "https://mocki.io/v1/af7d1b29-da7d-4eba-977b-cdb5caeb2fff";
```

Reply:
I did not know it was expressed this way. Thanks for pointing it out.
[01c3283](https://github.com/seeeen044/JavaScript_handson/pull/19/commits/01c3283081c978e3ec66cb93bb83b47161f2f8e8)